Code/Editor/MainWindow.cpp

```diff
@@ -709,13 +709,11 @@ void MainWindow::InitActions()
         .SetReserved()
         .SetStatusTip(tr("Undo last operation"))
         //.SetMenu(new QMenu("FIXME"))
```

Comment:
nit: would removing these lines effectively fulfill their desire of being fixes?

Reply:
I can do that too 😝